protobuf-kafkaconnect-converter/src/test/java/com/amazonaws/services/schemaregistry/kafkaconnect/protobuf/ToConnectTestDataGenerator.java

```diff
@@ -28,6 +33,9 @@
 
 public class ToConnectTestDataGenerator {
 
+    private static final String PROTOBUF_TAG = "protobuf.tag";
```

Comment:
Can we use the constants we already defined?
```
import static com.amazonaws.services.schemaregistry.kafkaconnect.protobuf.fromconnectschema.ProtobufSchemaConverterConstants.PROTOBUF_TAG;
import static com.amazonaws.services.schemaregistry.kafkaconnect.protobuf.fromconnectschema.ProtobufSchemaConverterConstants.PROTOBUF_TYPE;
```

Reply:
Yes, will fix